client/src/components/RestaurantInfo.jsx

```diff
@@ -4,7 +4,15 @@ import RestaurantInfoEntry from './RestaurantInfoEntry.jsx';
 import MapContainer from './MapContainer.jsx';
 
 const RestaurantInfo = props => (
+<<<<<<< HEAD
+<<<<<<< HEAD
   <div id="restaurantInfo" className="container">
+=======
+  <div id="restaurantInfo" className="container hidden">
+>>>>>>> Implemented "View All Button" toggle and style
```

Comment:
If this is a comment/placeholder text, are you planning on removing it in a later commit?